packages/linux-firmware/linux-firmware.spec

```diff
@@ -53,10 +49,471 @@ mkdir -p %{buildroot}/%{fwdir}/updates
+%exclude %{fwdir}/wil6210*.zst
+%exclude %{fwdir}/wsm_22.bin.zst
+%exclude %{fwdir}/xe
+%exclude %{fwdir}/yam
+%exclude %{fwdir}/yamaha
+
+# === Specific file pattern excludes ===
+
+# Exclude Intel WiFi firmware
+%exclude %{fwdir}/iwlwifi-*.ucode.zst
+%exclude %{fwdir}/iwlwifi-*.pnvm.zst
+
+# Exclude other wireless firmware
+%exclude %{fwdir}/ar*.fw.zst
+%exclude %{fwdir}/ar*.bin.zst
+%exclude %{fwdir}/ath3k-1.fw.zst
+%exclude %{fwdir}/htc_*.fw.zst
+%exclude %{fwdir}/carl9170-1.fw.zst
+%exclude %{fwdir}/mt*.bin.zst
+%exclude %{fwdir}/lbtf_usb.bin.zst
+%exclude %{fwdir}/lgs8g75.fw.zst
+%exclude %{fwdir}/lt9611uxc_fw.bin.zst
+
+# Exclude misc firmware
+%exclude %{fwdir}/as102_*.hex.zst
+%exclude %{fwdir}/bmi260-init-data.fw.zst
+%exclude %{fwdir}/cbfw-*.bin.zst
+%exclude %{fwdir}/ct2fw-*.bin.zst
+%exclude %{fwdir}/ctfw-*.bin.zst
+%exclude %{fwdir}/ctefx.bin.zst
+%exclude %{fwdir}/ctspeq.bin.zst
+%exclude %{fwdir}/cs42l43.bin.zst
+%exclude %{fwdir}/dvb*.fw.zst
+%exclude %{fwdir}/dvb*.inp.zst
+%exclude %{fwdir}/f2255usb.bin.zst
+%exclude %{fwdir}/hfi1_*.fw.zst
+%exclude %{fwdir}/isdbt_*.inp.zst
+%exclude %{fwdir}/agere_*.bin.zst
+%exclude %{fwdir}/tsse_firmware.bin.zst
+%exclude %{fwdir}/cmmb_*.inp.zst
+
+# Exclude TAS and TIAS firmware
+%exclude %{fwdir}/TAS*.bin.zst
+%exclude %{fwdir}/TIAS*.bin.zst
+%exclude %{fwdir}/TXNW*.bin.zst
+%exclude %{fwdir}/INT8866RCA2.bin.zst
+
+# Exclude additional firmware patterns
+%exclude %{fwdir}/a300_*.fw.zst
+
+# Exclude new Intel firmware directories and files
+%exclude %{fwdir}/intel/fw_sst_0f28.bin-48kHz_i2s_master.zst
+%exclude %{fwdir}/intel/avs
+%exclude %{fwdir}/intel/catpt
+%exclude %{fwdir}/intel/dsp_fw_*.bin.zst
+%exclude %{fwdir}/intel/fw_sst_*.bin.zst
+%exclude %{fwdir}/intel/ibt-*.ddc.zst
+%exclude %{fwdir}/intel/ibt-*.sfi.zst
+%exclude %{fwdir}/intel/ibt-*.bseq.zst
+%exclude %{fwdir}/intel/ice/ddp-lag
+%exclude %{fwdir}/intel/ipu
+%exclude %{fwdir}/intel/ish
+%exclude %{fwdir}/intel/iwlwifi
+%exclude %{fwdir}/intel/qat
+%exclude %{fwdir}/intel/vpu
+%exclude %{fwdir}/intel/vsc
+%exclude %{fwdir}/intel/IntcSST2.bin.zst
+%exclude %{fwdir}/intel/ipu3-fw.bin.zst
+%exclude %{fwdir}/intel/irci_*.bin.zst
+
+# ============================================================================
+# EXPLICIT INCLUDES: List of firmware files we DO want to ship
+# ============================================================================
+# After excluding unwanted firmware above, we now explicitly list every file
+# we want to include in the package. This ensures we only ship what we need.
+
+# Root-level firmware files
+%{fwdir}/myri10ge_eth_big_z8e.dat.zst
+%{fwdir}/myri10ge_eth_z8e.dat.zst
+%{fwdir}/myri10ge_ethp_big_z8e.dat.zst
+%{fwdir}/myri10ge_ethp_z8e.dat.zst
+%{fwdir}/myri10ge_rss_eth_big_z8e.dat.zst
+%{fwdir}/myri10ge_rss_eth_z8e.dat.zst
+%{fwdir}/myri10ge_rss_ethp_big_z8e.dat.zst
+%{fwdir}/myri10ge_rss_ethp_z8e.dat.zst
+%{fwdir}/phanfw.bin.zst
+
+# Network firmware - Broadcom NetXtreme II (bnx2x)
```

Comment:
Move the specific directories - `%dir %{fwdir}/bnx2x` - to their corresponding sections.